src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -49,7 +56,12 @@ private static GraphQLDocument Parse(string document)
 
         private ISchema BuildSchemaFrom(GraphQLDocument document)
         {
-            var schema = new Schema(DependencyResolver);
+            if (Directives.Any())
+            {
+                Visitors.Add(new DirectiveVisitorSelector(Directives, t => (SchemaDirectiveVisitor)ServiceProvider.GetService(t)));
```

Comment:
That is, there will now appear a general mechanism for expanding behavior via visitors/selectors, which, if desired, can be applied not only to directives? Cool.

Reply:
This is also place where `GetRequiredService` should be better than `GetService`.